### ISSUE TITLE
I can't reproduce the results in the paper.

### ISSUE THREAD

**Issue by cswjl** [UCSC-REAL/cifar-10-100n#8]
For CIFAR-10N worst, the result in the paper was 77%acc by CE loss, and I only got 67%acc using your code. In addition, can you give me experimental hyperparameters related to PES (Semi). Thank you.

**Comment by gordon-lim** [comment]
Could you share your solution?

**Comment by cswjl** [comment]
> Could you share your solution?

1. This paper tests the best acc, not the last acc. 2. The paper write using steplr at 50 ep, this is a typo. You should use steplr at 60 ep, see the code.

**Comment by gordon-lim** [comment]
1. For `python main.py --noise_type rand1 --is_human`, I got the best test accuracy of 90.3% at epoch 61. And for `python main.py --noise_type rand1` (synthetic), I got the best test accuracy of 90.7% at epoch 62. 
2. I used the provided code which does do the learning rate update at epoch 60.

I take it that you were able to reproduce their result then after having taken these 2 points into account? Strange.
